ONLY FOR MULTILIB RISCV!
Handles the extra / in the libdir... -dilfridge

diff -ruN Python-3.6.8.orig/Modules/getpath.c Python-3.6.8/Modules/getpath.c
--- Python-3.6.8.orig/Modules/getpath.c	2018-12-23 22:37:14.000000000 +0100
+++ Python-3.6.8/Modules/getpath.c	2019-04-21 01:05:35.127440301 +0200
@@ -796,6 +796,7 @@
     if (pfound > 0) {
         reduce(prefix);
         reduce(prefix);
+        reduce(prefix);
         /* The prefix is the root directory, but reduce() chopped
          * off the "/". */
         if (!prefix[0])
@@ -808,6 +809,7 @@
         reduce(exec_prefix);
         reduce(exec_prefix);
         reduce(exec_prefix);
+        reduce(exec_prefix);
         if (!exec_prefix[0])
                 wcscpy(exec_prefix, separator);
     }
